design-system/Text/Text.tsx

```diff
@@ -10,6 +10,7 @@ export const Text = React.forwardRef<RNText, ITextProps>((props, ref) => {
   const {
     weight,
     size,
+    color,
```

Comment:
cc @thierryskoda how would you like to handle these?
I think it's expected to instead be a preset? Or am I misunderstanding?

Reply:
I told Noe that it's okay for primary styling properties to also be props. Ideally we don't want too many and we should always look if the text we're adding is a preset but if not but we still need a custom color I think it's fine to have `weight` , `size` and  `color` as props